src/chocolatey.resources/helpers/functions/Get-ChocolateyUnzip.ps1

```diff
@@ -80,17 +80,11 @@ param(
   [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
 )
   $zipfileFullPath=$fileFullPath
-  if ($specificfolder) {
-    $fileFullPath=join-path $fileFullPath $specificFolder
-  }
 
   Write-FunctionCallLogMessage -Invocation $MyInvocation -Parameters $PSBoundParameters
 
   if ($packageName) {
     $packagelibPath = $env:ChocolateyPackageFolder
-    if (!(Test-Path -path $packagelibPath)) {
-      New-Item $packagelibPath -type directory
-    }

```

Comment:
Rob - It has been too long - I can't remember.  I am pretty sure that I wouldn't have removed it only because I noticed the folder was already there because I wouldn't know if under other conditions it might not exist by this point.

But it does seem like seem like an out of place removal.